renovate.json

```diff
@@ -1,25 +1,16 @@
 {
-  "dependencyDashboard": true,
-  "extends": [
-    "config:base"
-  ],
+  "extends": ["config:base", "group:allNonMajor", "schedule:weekly"],
   "labels": ["Maintenance 🔨"],
   "packageRules": [
     {
+      "groupName": "internal dependencies",
+      "groupSlug": "internal",
       "packagePatterns": [
-        "*"
+        "^@canonical",
+        "^canonicalwebteam",
+        "^vanilla-framework"
       ],
-      "excludePackagePatterns": [
-        "@canonical/react-components",
-        "jquery",
-        "prettier",
-        "vanilla-framework"
-      ],
```

Comment:
I think we probably still want to exclude these as they've all caused breakages at minor versions.

Reply:
done